src/main/java/jenkins/plugins/office365connector/FactsBuilder.java

```diff
@@ -104,7 +116,44 @@ public void addRemarks(List<Cause> causes) {
         facts.add(new Fact(NAME_REMARKS, causesStr.toString()));
     }
 
-    public void addCulprits(Result result, Set<User> authors) {
+    public void addScmDetails() {
```

Comment:
my intention was to have this class only for facts and now we moved lot of logic which is not responsible for facts here - this is not responsible of this class

Reply:
I agree, was just an suggestion 👍

Reply:
reverted the move of scm details :)